manifests/params.pp

```diff
@@ -268,6 +268,7 @@
   $server_startsnmptrapper                  = '0'
   $server_starttimers                       = '1'
   $server_starttrappers                     = '5'
+  $server_startpreprocessors                = '3'
```

Comment:
I think only 1 to 1000 is allowed? In that case can you please use an integer and the `Integer[1,1000]` datatype?

Reply:
That would be correct.

The parameter is also not mandatory, so I'm going to switch it to Optional, with a value of undef, and add a conditional for the value in the template.